### PR TITLE
fix(tsup): add multiple entry

### PR DESCRIPTION
**Pull request by manudeli**
related #101 

# Add multiple entry

support multiple entry

## AS IS

```ts
export const options: Options = {
  banner: { js: '"use client"' },
  format: ['cjs', 'esm'],
  entry: ['src/index.ts'],
  sourcemap: true,
  clean: true,
  dts: true,
}
```

image

## TO BE

```ts
export const options: Options = {
  banner: { js: '"use client"' },
  format: ['cjs', 'esm'],
  entry: ['src/index.ts', 'src/*.ts', 'src/*.tsx'],
  sourcemap: true,
  clean: true,
  dts: true,
  splitting: false,
}
```

image

## PR Checklist

- [x] I have written documents and tests, if needed.

### CODE REVIEW

File: packages/react-query/package.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
   "module": "dist/index.mjs",
   "types": "dist/index.d.ts",
   "scripts": {
-    "build": "tsup src/index.ts",
+    "build": "tsup",
```
Comment (manudeli) [inline packages/react-query/package.json:29]: Moved all options to @suspensive/tsup

File: packages/react/src/Delay.tsx [hunk 2]
```diff
@@ -1,5 +1,3 @@
-'use client'
```
Comment (manudeli) [inline packages/react/src/Delay.tsx:1]: all directive to @suspensive/tsup's banner

**Review by manudeli — COMMENTED** [review]
I leave comments as my intention

**Review by okinawaa — APPROVED** [review]

File: packages/react-query/package.json (hunk 1 above)
Comment (okinawaa) [reply packages/react-query/package.json:29]: cool

**Pull request merged**